src/App.vue

```diff
@@ -9,18 +9,40 @@
         {{ $t('app.routerTitle.register') }}
       </router-link>
 
-        <router-link to="/login">
+      <router-link to="/login">
         {{ $t('app.routerTitle.login') }}
       </router-link>
-      
+
       <router-link to="/reset-password">
         {{ $t('general.resetPassword.title') }}
       </router-link>
       <router-link to="/forgot-password">
         {{ $t('general.forgotPassword.title') }}
       </router-link>
-
+      <router-link to="/dashboard">
+        Dashboard
+      </router-link>
+      <button @click="logOut">Log out</button>
     </div>
     <router-view />
   </div>
-</template>
\ No newline at end of file
+</template>
+
+<script>
+// import store from '@/store/store';
+
+export default {
+  computed: {
+    getLoggedInState: function() {
+      return this.$store.getters.userState.loggedIn;
+    },
+  },
+  methods: {
+    logOut() {
+      this.$store.commit('setUserState');
```

Comment:
Here is an example of how we should use the getters and actions  from vuex 
![image](https://user-images.githubusercontent.com/9899327/130564311-a1b295f8-6f37-4328-a574-ce4822314e57.png)